src/main/resources/view/MainWindow.fxml

```diff
@@ -46,7 +46,7 @@
           </padding>
         </StackPane>
 
-        <VBox fx:id="personList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
+        <VBox fx:id="transactionList" styleClass="pane-with-border" minWidth="340" prefWidth="340" VBox.vgrow="ALWAYS">
           <padding>
             <Insets top="10" right="10" bottom="10" left="10" />
           </padding>
```

Comment:
In line 53, `"personListPanelPlaceholder"` should be changed to `"transactionListPanelPlaceholder"`.